src/sentry/static/sentry/app/components/events/interfaces/threads/threadSelector/filterThreadInfo.tsx

```diff
@@ -2,23 +2,18 @@ import {Thread, Frame} from 'app/types/events';
 import {Event} from 'app/types';
 import {trimPackage} from 'app/components/events/interfaces/frame/utils';
 
-import getThreadStacktrace from './getThreadStacktrace';
-import getRelevantFrame from './getRelevantFrame';
-import trimFilename from './trimFilename';
-
-// TODO(i18n): add traslations here
-const NOT_FOUND_FRAME = '<unknown>';
+import {getThreadStacktrace} from './getThreadStacktrace';
+import {getRelevantFrame} from './getRelevantFrame';
+import {trimFilename} from './trimFilename';
```

Comment:
I would suggest not changing this because it's going to add to the inconsistency with our import/exports -- currently we generally use default exports for everything unless a file exports multiple things